src/parser/shared/modules/Channeling.js

```diff
@@ -1,6 +1,7 @@
 import Analyzer from 'parser/core/Analyzer';
 import EventEmitter from 'parser/core/modules/EventEmitter';
 import CASTS_THAT_ARENT_CASTS from 'parser/core/CASTS_THAT_ARENT_CASTS';
+import SPELLS from 'common/SPELLS/index';
```

Comment:
This import should not include the index part, this is implicit

Reply:
Ok I'll remember that for next time, thanks!